.npmrc

```diff
@@ -0,0 +1,2 @@
+@mavenagi:registry=https://registry.npmjs.org/
```

Comment:
do we need this? 
can the shared package just be public?

Reply:
this is why the vercel deployment failed btw

Reply:
Yes I pinged @bsadecki-magi about this, I think we can just make the dependency public now

Reply:
We're public now 🥳